plugins/colorschemes/base16/default.nix

```diff
@@ -0,0 +1,118 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+# We configure this plugin manually (no `settings` option) so there is no point in using
+# `mkNeovimPlugin` here.
+  helpers.vim-plugin.mkVimPlugin config {
+    name = "base16";
+    isColorscheme = true;
+    originalName = "base16.nvim";
+    defaultPackage = pkgs.vimPlugins.base16-nvim;
+
+    maintainers = [maintainers.GaetanLepage];
+
+    # We manually set the colorscheme if needed.
+    colorscheme = null;
+
+    # TODO introduced 2024-03-12: remove 2024-05-12
+    imports = let
+      basePluginPath = ["colorschemes" "base16"];
+    in [
+      (
+        mkRenamedOptionModule
+        (basePluginPath ++ ["customColorScheme"])
+        (basePluginPath ++ ["colorscheme"])
+      )
+      (
+        mkRenamedOptionModule
+        (basePluginPath ++ ["useTruecolor"])
+        ["termguicolors"]
```

Comment:
isn't this `["options" "termguicolors"]` ? Our very unfortunate mapping to neovim options...